src/ontology/mondo-ingest.Makefile

```diff
@@ -187,6 +190,16 @@ config/term_exclusions.txt config/exclusion_reasons.robot.template.tsv: config/o
 	cat config/*_term_exclusions.txt > config/term_exclusions.txt; \
 	awk '(NR == 1) || (NR == 2) || (FNR > 2)' config/*_exclusion_reasons.robot.template.tsv > config/exclusion_reasons.robot.template.tsv
 
+# todo: Is it desired for there to be merged `.ttl` and `.obo` files? How to do that? Pass multiple `--input` and use the merged `config/exclusion_reasons.robot.template.tsv` for --template?
+# TODO: add back these prerequisites to this target:
+# config/%_exclusion_reasons.ttl: mirror/%.owl config/%_exclusion_reasons.robot.template.tsv
+config/%_exclusion_reasons.ttl:
```

Comment:
@matentzn 
### Success cases
DOID, OMIM, ICD10WHO

### Failures
#### ORDO
```
sh run.sh make MIR=false config/ordo_exclusion_reasons.ttl
robot template --input mirror/ordo.owl --template config/ordo_exclusion_reasons.robot.template.tsv --output config/ordo_exclusion_reasons.ttl
UNKNOWN ENTITY ERROR could not interpret 'Orphanet:158048' in row 3, column 1 ("ID") in table "config/ordo_exclusion_reasons.robot.template.tsv".
```

#### ICD10CM
```
sh run.sh make MIR=false config/icd10cm_exclusion_reasons.ttl
robot template --input mirror/icd10cm.owl --template config/icd10cm_exclusion_reasons.robot.template.tsv --output config/icd10cm_exclusion_reasons.ttl
UNKNOWN ENTITY ERROR could not interpret 'ICD10CM:P90-P96' in row 3, column 1 ("ID") in table "config/icd10cm_exclusion_reasons.robot.template.tsv".
```

#### NCIT
```
sh run.sh make MIR=false config/ncit_exclusion_reasons.ttl
robot template --input mirror/ncit.owl --template config/ncit_exclusion_reasons.robot.template.tsv --output config/ncit_exclusion_reasons.ttl
ANNOTATION PROPERTY ERROR could not handle annotation property: MONDOREL:has_exclusion_reason
```

Reply:
They should all work now. Re-run.